api/swagger.yaml

```diff
@@ -243,6 +243,9 @@ definitions:
       ReadOnly:
         description: "Whether the mount should be read-only."
         type: "boolean"
+      Consistency:
+        description: "The consistency requirement for the mount: `default`, `consistent`, `cached`, or `delegated`."
```

Comment:
Question; if this field is left empty, is that equal to "default"? Wondering if we actually _need_ the literal "default" value